### ISSUE TITLE
Can not pip install 3.7.1 with pip 1.5.4

### ISSUE THREAD

**Issue by aquilax** [HypothesisWorks/hypothesis-python#558]
Getting an error while installing `3.7.1` with pip:

```bash
(env)vagrant@vagrant-ubuntu-trusty-64:~$ pip install hypothesis==3.7.1
Downloading/unpacking hypothesis==3.7.1
  Downloading hypothesis-3.7.1.tar.gz (79kB): 79kB downloaded
  Running setup.py (path:/home/vagrant/env/build/hypothesis/setup.py) egg_info for package hypothesis
    error in hypothesis setup command: 'extras_require' must be a dictionary whose values are strings or lists of strings containing valid project/version requirement specifiers.(env)vagrant@vagrant-ubuntu-trusty-64:~$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 14.04.5 LTS
Release:        14.04
Codename:       trusty
    Complete output from command python setup.py egg_info:
    error in hypothesis setup command: 'extras_require' must be a dictionary whose values are strings or lists of strings containing valid project/version requirement specifiers.

----------------------------------------
Cleaning up...
Command python setup.py egg_info failed with error code 1 in /home/vagrant/env/build/hypothesis
Storing debug log for failure in /home/vagrant/.pip/pip.log
(env)vagrant@vagrant-ubuntu-trusty-64:~$ python --version
Python 3.4.3
(env)vagrant@vagrant-ubuntu-trusty-64:~$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 14.04.5 LTS
Release:        14.04
Codename:       trusty
(env)vagrant@vagrant-ubuntu-trusty-64:~$ pip -V
pip 1.5.4 from /home/vagrant/env/lib/python3.4/site-packages (python 3.4)
```
Details from the log:
```bash
Cleaning up...
  Removing temporary dir /home/vagrant/env/build...
Command python setup.py egg_info failed with error code 1 in /home/vagrant/env/build/hypothesis
Exception information:
Traceback (most recent call last):
  File "/home/vagrant/env/lib/python3.4/site-packages/pip/basecommand.py", line 122, in main
    status = self.run(options, args)
  File "/home/vagrant/env/lib/python3.4/site-packages/pip/commands/install.py", line 278, in run
    requirement_set.prepare_files(finder, force_root_egg_info=self.bundle, bundle=self.bundle)
  File "/home/vagrant/env/lib/python3.4/site-packages/pip/req.py", line 1229, in prepare_files
    req_to_install.run_egg_info()
  File "/home/vagrant/env/lib/python3.4/site-packages/pip/req.py", line 325, in run_egg_info
    command_desc='python setup.py egg_info')
  File "/home/vagrant/env/lib/python3.4/site-packages/pip/util.py", line 697, in call_subprocess
    % (command_desc, proc.returncode, cwd))
pip.exceptions.InstallationError: Command python setup.py egg_info failed with error code 1 in /home/vagrant/env/build/hypothesis
(env)vagrant@vagrant-ubuntu-trusty-64:~$
(env)vagrant@vagrant-ubuntu-trusty-64:~$ pip install hypothesis==3.7.1
Downloading/unpacking hypothesis==3.7.1
  Downloading hypothesis-3.7.1.tar.gz (79kB): 79kB downloaded
  Running setup.py (path:/home/vagrant/env/build/hypothesis/setup.py) egg_info for package hypothesis
    error in hypothesis setup command: 'extras_require' must be a dictionary whose values are strings or lists of strings containing valid project/version requirement specifiers.
    Complete output from command python setup.py egg_info:
    error in hypothesis setup command: 'extras_require' must be a dictionary whose values are strings or lists of strings containing valid project/version requirement specifiers.
```
Version 3.7.0 installs without any issue:
```bash
(env)vagrant@vagrant-ubuntu-trusty-64:~$ pip install hypothesis==3.7.0
Downloading/unpacking hypothesis==3.7.0
  Downloading hypothesis-3.7.0.tar.gz (79kB): 79kB downloaded
  Running setup.py (path:/home/vagrant/env/build/hypothesis/setup.py) egg_info for package hypothesis

Installing collected packages: hypothesis
  Found existing installation: hypothesis 3.5.0
    Uninstalling hypothesis:
      Successfully uninstalled hypothesis
  Running setup.py install for hypothesis

Successfully installed hypothesis
Cleaning up...
```

**Comment by DRMacIver** [comment]
Yup, sorry about that. Fix is on the way in #557. Should be live in the next hour or so.

**Comment by aquilax** [comment]
Awesome. Closing the issue.
